integration/lsp/smoke_spec.ts

```diff
@@ -108,7 +108,8 @@ describe('Angular Language Service', () => {
           'completionProvider':
               {'resolveProvider': false, 'triggerCharacters': ['<', '.', '*', '[', '(', '$', '|']},
           'definitionProvider': true,
-          'hoverProvider': true
+          'hoverProvider': true,
+          'workspaceFolders': {'supported': true}
```

Comment:
This needs to be wrapped in `workspace`:
```suggestion
          'workspace': {
              'workspaceFolders': {'supported': true}
          }
```